clients/cli/src/main.rs

```diff
@@ -82,6 +86,13 @@ enum Command {
     },
     /// Clear the node configuration and logout.
     Logout,
+    /// Hidden command for subprocess proof generation
+    #[command(hide = true, name = "prove-fib-subprocess")]
+    ProveFibSubprocess {
```

Comment:
I'm confused about what this is for, if it's not meant to be used. Does it need to be committed?